dev/NavigationView/NavigationView_ApiTests/NavigationViewTests.cs

```diff
@@ -1091,5 +1091,42 @@ public void VerifyHierarchicalNavigationTopModeMenuItemsSourceDoesNotCrash()
                 Content.UpdateLayout();
             });
         }
+
+        [TestMethod]
+        public void VerifyNavigationViewItemToolTip()
```

Comment:
If this test will be left as is, it should probably be named "VerifyNavigationViewItemToolTipCollapsedPane" since you are only verifying which tooltips render when the pane is collapsed, not if no tooltips will be rendered when pane is expanded or what the behavior for DIsplayMode=Top is.

Reply:
I am currently rewriting the introduced API test which hopefully addresses some of the issues/comments here. Not sure if the commit will make it out today but it should be done by tomorrow.

Reply:
It took a bit longer but here are the updated API tests:
* VerifyNavigationViewItemToolTipCreation
This test only cares about the creation of a tooltip for a NavigationViewItem based on its content or if custom set. For example, it verifies that an empty item content (null or empty string) does not create a tooltip.

* VerifyNavigationViewItemToolTipPaneDisplayMode
This test verifies the relation between tooltips and the pane display modes. It verifies that tooltips will be shown in the display modes required and otherwise not.